jwk/manager_sql.go

```diff
@@ -211,6 +211,13 @@ func (m *SQLManager) DeleteKey(ctx context.Context, set, kid string) error {
 	return nil
 }
 
+func (m *SQLManager) DeleteOldKeys(ctx context.Context, set string, date time.Time) error {
```

Comment:
The manager_test should have a test for this method

Reply:
This is also I will do it!